### ISSUE TITLE
Hhi Ziyi

### ISSUE THREAD

**Issue by feilong0309** [Wang-ZiYi/streamGraphSample#1]
I also do the research on some graph staff. I am wondering that  which methods do you want to implement. If I do not know the exact number of subgraph, how should you do?

**Comment by chengjue-2445** [comment]
I want to implement streaming NS, streaming ES, optimized PIES and PIES-min now and I'm not sure what else I will do in the future. You can read the paper "Space-efficient sampling from social activity streams" and the review "Network Sampling From Static to Streaming Graphs".

And the number of subgraph actually is about 30% of the number of the origin graph, and the ratio can be 10%-50%. And I'm sorry that I do not know how to get the number of origin graph if the origin graph is an unlimited stream graph.

By the way, I will not update this project on Wang-Ziyi but on hustustor, https://github.com/hustustor/streamGraphSample.git.

**Comment by feilong0309** [comment]
Thanks for your comments, I have read the paper network sampling from static to streaming graph. Quite interesting sampling algorithms. Do you have finished the PIES algorithm, maybe we can discuss it if possible.

**Comment by chengjue-2445** [comment]
Yes I have finished the PIES algorithm and want to do some improvement but havent pushed the update on the GIT yet.

**Comment by feilong0309** [comment]
Dear Ziyi,

Great, would you mind to send me one copy of your developing code, I want
to join in developing the streaming sampling algorithm?

Cheers,
Jianpeng

**Comment by chengjue-2445** [comment]
I created a new repository called pies and upload part of my codes there,

**Comment by feilong0309** [comment]
OK, thanks.
